get-deps

```diff
@@ -51,12 +51,14 @@ if ! check_rust_version ; then
   exit 1
 fi
 
+SUDO=sudo
```

Comment:
I'd suggest doing something like this, which should allow the caller to specify what `SUDO` expands to, otherwise, only sets it if sudo is present in the path.  That might be sufficient to fix the human use case and the CI use case

```
if test -z "${SUDO+x}" ; then
   if hash sudo 2>/dev/null ; then
      SUDO="sudo"
   fi
fi
```

Reply:
Thanks. I've updated the PR. Please take a look.